From a8fe5e43261a33657a123f52f6d2886eacfe08ef Mon Sep 17 00:00:00 2001
From: Sathishkumar Muruganandam <quic_murugana@quicinc.com>
Date: Thu, 18 Jan 2024 16:31:29 +0530
Subject: [PATCH] hostapd: allow hidden ssid bring-up with mbssid config

Remove the hidden ssid bring-up restriction with MBSSID case.

Signed-off-by: Sathishkumar Muruganandam <quic_murugana@quicinc.com>
---
 src/ap/ap_config.c | 6 ------
 1 file changed, 6 deletions(-)

--- a/src/ap/ap_config.c
+++ b/src/ap/ap_config.c
@@ -1520,12 +1520,6 @@ static int hostapd_config_check_bss(stru
 	}
 #endif /* CONFIG_IEEE80211BE */
 
-	if (full_config && bss->ignore_broadcast_ssid && conf->mbssid) {
-		wpa_printf(MSG_ERROR,
-			   "Hidden SSID is not suppored when MBSSID is enabled");
-		return -1;
-	}
-
 	return 0;
 }
 
